test_rclcpp/test/test_timer.cpp

```diff
@@ -59,7 +59,7 @@ TEST(CLASSNAME(test_time, RMW_IMPLEMENTATION), timer_fire_regularly) {
     printf("spin_node_some() for 4s\n");
     while (std::chrono::steady_clock::now() < start + 4.5 * period) {
       executor.spin_node_some(node);
-      std::this_thread::sleep_for(period / 25);
```

Comment:
Good point. But who wants to maintain that string. Now it says "for some time" :wink: